src/lib/utils/references.ts

```diff
@@ -57,7 +58,7 @@ export const platformMap: Record<Language | string, string> = {
     [Platform.ClientRest]: 'REST',
     [Platform.ServerDart]: 'Dart',
     [Platform.ServerDeno]: 'Deno',
-    [Platform.ServerDotNet]: '.NET',
+    [Platform.ServerDotNet]: '.NET (beta)',
```

Comment:
Please make sure the docs still work. dotnet works in production, but didn't when I tested the PR. It showed:

![image](https://github.com/user-attachments/assets/db8f7d59-c4cb-43bc-806e-ca1c30268a84)